spark/src/main/resources/error/delta-error-classes.json

```diff
@@ -1079,6 +1085,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "DELTA_MATERIALIZED_ROW_ID_COLUMN_NAME_MISSING" : {
```

Comment:
only a message row ids, not for commit versions?

Reply:
Good catch, I changed the error to correctly cover row commit version